### ISSUE TITLE
Rank 6 Combat Skills: In Need of a Revamp

### ISSUE THREAD

**Issue by somethingfinite** [Arx-Game/arxcode#300]
**Is your feature request related to a problem? Please describe.**

Combat 6s as they currently stand are an insane amount of XP investment for a very minimal return - I feel as though they once served a function, but that function only exists in a legacy capacity rather than as a viable or even passing realistic goal. 

In essence, if you don't already have a Combat 6 (Read: From the moment in time when they were less costly to realistically obtain), you're _exceedingly unlikely to ever obtain one_, which sets a dangerous precedent of legacy power gate-keeping.

The heart of the issue: The baseline is way too expensive to get a Combat 6 as it currently stands, to the point that it's wildly impractical to ever want to invest in them if you haven't already done so.

For example, my current XP Cost for one: ~4600

My current lifetime XP - 1897, and that's with a great deal of hitting the exp leaderboard over the course of my year here.

Generously, it's going to take someone who's as new as me about 2-2.5 years to afford a combat 6, if they never, ever spend a point in anything else. It's even worse for other players. I believe when this came up in systems, someone said that their current tax had them up to 6.5k in order to afford one. While this makes sense with the tax system, it also creates an insane amount of time where you character is not meaningfully reaping any benefit from their experience.

Now, putting myself into the shoes of that player, that balloons up to about 3-4 years of saving all of your exp in order to obtain a 6. Let's forget all about the mechanics of it for just a moment: **That is an extremely unfun thing to do, and should by itself be a reason to review the costs.**

But hey, mechanics. There's plenty of that to go around, too. Let's say that I decided, "Hey. You know, what? I really like the idea of getting that six in dodge, but I can't justify it to myself to cut my exp spending off at the knees for that long."

Enter Knacks.

Some quick and dirty knack math:

```
Knack Rank | Cost | Total Cost | Bonus Crit | Effective Crit
1 | 150 | 150 | 1 | 6
2 | 70 | 220 | 1.5 | 6.5
3 | 80 | 300 | 2 | 7
4 | 90 | 390 | 2.5 | 7.5
5 | 100 | 490 | 3 | 8
6 | 110 | 600 | 3.5 | 8.5
7 | 120 | 720 | 4 | 9
8 | 130 | 850 | 4.5 | 9.5
9 | 140 | 990 | 5 | 10
10 | 150 | 1140 | 5.5 | 10.5
11 | 160 | 1300 | 6 | 11
12 | 170 | 1470 | 6.5 | 11.5
13 | 180 | 1650 | 7 | 12
14 | 190 | 1840 | 7.5 | 12.5
15 | 200 | 2040 | 8 | 13
16 | 210 | 2250 | 8.5 | 13.5
17 | 220 | 2470 | 9 | 14
18 | 230 | 2700 | 9.5 | 14.5
19 | 240 | 2940 | 10 | 15
20 | 250 | 3190 | 10.5 | 15.5
21 | 260 | 3450 | 11 | 16
22 | 270 | 3720 | 11.5 | 16.5
23 | 280 | 4000 | 12 | 17
24 | 290 | 4290 | 12.5 | 17.5
25 | 300 | 4590 | 13 | 18
26 | 310 | 4900 | 13.5 | 18.5
27 | 320 | 5220 | 14 | 19
28 | 330 | 5550 | 14.5 | 19.5
29 | 340 | 5890 | 15 | 20
30 | 350 | 6240 | 15.5 | 20.5
```

Me, as the player with a cost of 4553, could instead decide to invest in a knack. As this is a combat skill, for combat purposes, it's relatively easy to choose an ideal combat knack that replaces any and all need for a 6 for a similar amount of exp.

For example: I could buy a 6, which would give me an extra d10 and an extra keep. Always good. Numbers go up! But that entire time in the interim, I am getting absolutely nothing. And at the end of the day, buying ranks in knack gives incremental growth that mechanically eclipses the value of a 6 for the XP cost.

**Equivalent knack perks for Player A (4600):**

1d10 + 1 Keep

OR

Rank 25 in Knack: +25 flat increase prior to criticals, and 13% bonus chance to crit.

**Equivalent knack perks for Player B (6500)**

1d10 + 1 Keep

OR

Rank 30 in Knack: +30 flat increase prior to criticals, and 15% bonus chance to crit.

Now, we can debate whether or not people would prefer it one way or another, but the major point in Knack's favor is that you're not saving up. You're not tied down. You can detour to spend on something else without feeling like you're deviating from your goal. You're incrementally building towards something.

TLDR;

- Combat 6s used to be easier to acquire.

- They are not anymore. They are now ludicrously expensive. This leads to a divide (or perceived divide, just as bad) based on creation date that is not easily bridged.

- Worse still, they no longer make any sense at all mechanically to buy, because comparative knacks will give you more bang for your buck, all while letting you grow incrementally.

- The simple human psychology of saving up for 2-3 years while not touching your character at all is near-insurmountable, and frankly not very fun by design.

**Describe the solution (or alternatives) you'd like**

A few proposals that would make sense to me:

1. **Lower the cost of Combat 6s and call it a day** - Easy. Quick.
2. **Lower the cost of Combat 6s to a flat rate (untaxed) and restrict players to only having one Combat 6.** Period. This prevents people from going off the rails while still accumulating acclaim for the 6 they've chosen to acquire.
3. **Take a page out of knack's book and make 'checkpoints' towards a 6** that provide small bonuses to take some of the sting out of saving up for the foreseeable future of your character.
4. **Nuclear option: Refund existing Combat 6s and close them.** They're so disastrously unreachable that they only serve to demoralize newer players, anyhow and create the perception of a divide that can not realistically be closed - even if that is in perceived power alone, that can be a huge turnoff to people who are just starting out.

**Additional context**

_This bothers me on a system level far more than an issue of 'Well, Player A has a 6 and I don't and wah, so let's nip that one in the bud. Systems should be agnostic to when you started the game, and also meaningful to everyone. This is an instance where a system is rather flawed for anyone who **Is Not Old Enough To Ride.**')_

Feel free to discuss!

**Comment by TehomCD** [comment]
I don't think I'll touch any xp costs at all until we redo how earning/spending xp/training skills/stats/knacks/etc works. Doing it piecemeal leads into rabbitholes that are massive wastes of time like haggling.

**Comment by somethingfinite** [comment]
Was unaware that was a roadmapped consideration. Fair enough, then!
